app/jobs/elasticsearch_delete_by_query_job.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+# Deletes documents from the Elasticsearch index where "updated_at" field is older than the configured retention days
+# - Starts async delete_by_query and stores ES task id in Redis
+# - On each run: cancels previous task, and starts a new one
+# - Uses ActiveJob retry_on for transient start errors
+class ElasticsearchDeleteByQueryJob < ApplicationJob
+  queue_as :searchgov
+
+  DEFAULT_REQUESTS_PER_SECOND = 500
+  DEFAULT_SCROLL_SIZE = 5000
+  START_RETRY_ATTEMPTS = 3
+  REDIS_KEY_PREFIX = "elasticsearch_delete_by_query:task_id"
+
+  retry_on Elasticsearch::Transport::Transport::Errors::ServiceUnavailable,
+           Faraday::TimeoutError,
+           Errno::ETIMEDOUT,
+           wait: ->(executions) { 2**executions },
```

Comment:
What do you think about leaving the default `wait` behavior "first wait ~3s, then ~18s, then ~83s, etc" and retry attempts "5". Only specify the errors it will retry on.